database/seeders/CustomTourSeeder.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\CustomTour;
+use Illuminate\Database\Seeder;
+
+class CustomTourSeeder extends Seeder
+{
+    public function run(): void
+    {
+        // First Custom Tour
```

Comment:
Update these examples to be different according to content. E.g. "Tour with complete content", "Tour without a description" etc.

Reply:
Thanks, I've updated to only have two custom tours. One with complete content and the other with no description.